runtime/src/main/java/org/corfudb/util/FileWatcher.java

```diff
@@ -110,6 +124,18 @@ private void poll() {
         }
     }
 
+    private String getFileHash() {
+        try {
+            MessageDigest digest = MessageDigest.getInstance("SHA-256");
```

Comment:
Minor optional point: In DataStore.java we use the Hasher and here we are using message digest - should we just use one hashing algorithm?

Reply:
I agree that in general it's better to do the same thing with the same tool in a codebase. But I think SHA256 will be good here given that 1. SHA256 is more collision-safe than crc32, and 2. there is no existing utility in current codebase that can be directly used.